builder/remotecontext/git.go

```diff
@@ -21,9 +22,14 @@ func MakeGitContext(gitURL string) (builder.Source, error) {
 	}
 
 	defer func() {
-		// TODO: print errors?
-		c.Close()
-		os.RemoveAll(root)
+		err := c.Close()
+		if err != nil {
+			fmt.Println(err)
```

Comment:
I am not sure `fmt` is obvious here. Why not using `logrus` or thinking on a way to return an error so it will be printed somewhere else.

Reply:
`logrus` sounds appropriate